### PR TITLE
Youtube Hosted videos - add next video autoplay etc

### PR DESCRIPTION
**Pull request by lps88**
## What does this change?
- Add the autoplay next video functionality to youtube version of hosted video.
- Add the current time/duration below the video title
## Screenshots

(has act for wildlife next page hardcoded for now)
image

image
## Request for comment

@guardian/labs-beta @gidsg @akash1810

### CODE REVIEW

**Comment by kelvin-chappell** [comment]
👍

**Comment by Calanthe** [comment]
👍

**Comment by lps88** [comment]
@Calanthe @kelvin-chappell can you look over the more recent changes too? Ta!

File: commercial/app/views/hosted/guardianHostedVideo.scala.html [hunk 1]
```diff
@@ -47,10 +47,28 @@
...
+                                </div>
+                            </div>
+                            <div class="vjs-time-control vjs-time-divider" aria-hidden="true">
+                                <div>
+                                    <span>/</span>
+                                </div>
+                            </div>
+                            <div class="vjs-duration vjs-time-control vjs-control" aria-hidden="true">
+                                <div class="vjs-duration-display" aria-live="off">
+                                    <span class="vjs-control-text">Duration Time</span>@{"%2d:%02d".format(page.video.duration/60, page.video.duration % 60)}
```
Comment (akash1810) [inline commercial/app/views/hosted/guardianHostedVideo.scala.html]: would it be worth pulling this in to a property in its own right, to keep the template super simple?

**Review by akash1810 — APPROVED** [review]

File: commercial/app/views/hosted/guardianHostedVideo.scala.html (hunk 1 above)
Comment (lps88) [inline commercial/app/views/hosted/guardianHostedVideo.scala.html]: Yes good idea, @Calanthe is taking this over from here, she will be able to do it really easily  👍

Comment (Calanthe) [inline commercial/app/views/hosted/guardianHostedVideo.scala.html]: @akash1810 I just created a PR with that https://github.com/guardian/frontend/pull/14533

**Pull request merged**